drivers/serial/uart_stm32.c

```diff
@@ -308,145 +308,72 @@ static int uart_stm32_init(struct device *dev)
 	return 0;
 }
 
-
-#ifdef CONFIG_UART_STM32_PORT_1
-
-#ifdef CONFIG_UART_INTERRUPT_DRIVEN
-static void uart_stm32_irq_config_func_1(struct device *dev);
-#endif	/* CONFIG_UART_INTERRUPT_DRIVEN */
-
-static const struct uart_stm32_config uart_stm32_dev_cfg_1 = {
-	.uconf = {
-		.base = (u8_t *)CONFIG_UART_STM32_PORT_1_BASE_ADDRESS,
-#ifdef CONFIG_UART_INTERRUPT_DRIVEN
-		.irq_config_func = uart_stm32_irq_config_func_1,
-#endif	/* CONFIG_UART_INTERRUPT_DRIVEN */
-	},
+/* Define clocks */
 #ifdef CONFIG_CLOCK_CONTROL_STM32_CUBE
-	.pclken = { .bus = STM32_CLOCK_BUS_APB2,
-		    .enr = LL_APB2_GRP1_PERIPH_USART1 },
+	#define STM32Fxxx_CLOCK_UART(type, apb, n)			\
```

Comment:
Change macro name as this is also used for STM32LXX as well already.
STM32_CLOCK_UART would be enough

Then, use of dts should modify this as well anytime soon.
Hence I wonder if this is the right time to do this change.
@kumar, @Andy Gross, your view on this?